FF1Blazorizer/wwwroot/tooltips/tooltips.json

```diff
@@ -1260,7 +1266,7 @@
 		"Id": "formationShuffleModeDropDown",
 		"title": "Enemy Formation Shuffle Mode",
 		"screenshot": "formationShuffleModeDropDown.gif",
-		"description": "Shuffles the encounters that appear in enemy domains.\nVanilla: Enemy groups remain as they exist in the original game. This allows players to predict encounters in a dungeon if they know the vanilla values.\n Shuffle Rarity: Enemy groups are shuffled within a zone, so that Group 1 fights can appear as Group 6, etc. This removes the ability to 'scout' the encounter table and know which fights will appear without having to re-learn the process for each new dungeon.\nTiered Rarity Shuffle: Enemy formations will shuffle their encounter group based on their original rarity. (1-4 will tend to be lower in rarity, 5-6 can sometimes be lower in rarity, 7-8 will rarely be lower in rarity.)\n  Shuffle Across Zones: Shuffles the domains (such that Ice Cave groups could appear in Gurgu Volcano, etc.) but does not alter the contents of the domains themselves.\n Totally Random: Draws at random and populates the encounter table, excepting fiends, plot-triggered encounters, and empty encounters. These encounters can appear in any zone except the starting zones.\n Note: This feature is disabled when generating new enemy formations (via Enemizer)."
+		"description": "Shuffles the encounters that appear in enemy domains.\nVanilla: Enemy groups remain as they exist in the original game. This allows players to predict encounters in a dungeon if they know the vanilla values.\n Shuffle Rarity: Enemy groups are shuffled within a zone, so that Group 1 (common) fights can appear as Group 6 (uncommon), etc. This removes the ability to 'scout' the encounter table and know which fights will appear without having to re-learn the process for each new dungeon.\nTiered Rarity Shuffle: Enemy formations will shuffle their encounter group based on their original rarity. (1-4 will tend to be lower in rarity, 5-6 can sometimes be lower in rarity, 7-8 will rarely be lower in rarity.)\n  Shuffle Across Zones: Shuffles the domains (such that Ice Cave groups could appear in Gurgu Volcano, etc.) but does not alter the contents of the domains themselves.\n Totally Random: Draws at random and populates the encounter table, excepting fiends, plot-triggered encounters, and empty encounters. These encounters can appear in any zone except the starting zones.\n Note: This feature is disabled when generating new enemy formations (via Enemizer)."
```

Comment:
The tooltip for Shuffle Across Zones is wrong, all it does is take all encounters from all 128 zones and shuffle them, regardless of rarity.

Reply:
Ope, didn't see this comment until now - fixed.